packages/grid/x-grid-modules/src/components/styled-wrappers/grid-root.tsx

```diff
@@ -1,255 +1,22 @@
 import * as React from 'react';
-import styled from 'styled-components';
 import { GridOptions } from '../../models';
 import { classnames } from '../../utils';
+import { getStyles } from './getStyles';
```

Comment:
Why move the styles to a different file?

Reply:
it's just cleaner. One is the component using the style, the other file is the style